docs/REST API/rest-api.md

```diff
@@ -4721,7 +4725,8 @@ Add a configuration profile to enforce custom settings on macOS and Windows host
 | ------------------------- | -------- | ---- | ------------------------------------------------------------------------------------------------------------- |
 | profile                   | file     | form | **Required.** The .mobileconfig and JSON for macOS or XML for Windows file containing the profile. |
 | team_id                   | string   | form | _Available in Fleet Premium_. The team ID for the profile. If specified, the profile is applied to only hosts that are assigned to the specified team. If not specified, the profile is applied to only to hosts that are not assigned to any team. |
-| labels                    | array     | form | _Available in Fleet Premium_. An array of labels to filter hosts in a team (or no team) that should get a profile. |
```

Comment:
Note that the URL of this endpoint still documents the old deprecated one (see line 4720 above). The new endpoint is `POST /api/v1/fleet/configuration_profiles`.

Reply:
Sorry, just realized this was called out here: https://github.com/fleetdm/fleet/pull/18844/files#r1594207421. Marking as resolved.